CommanderConda/construct.yaml

```diff
@@ -10,14 +10,27 @@ company: "ParaTools, Inc."
 
 channels:
   - https://conda.anaconda.org/conda-forge
+  - https://conda.anaconda.org/plotly
+  - http://repo.anaconda.com/pkgs/main
+
+conda_default_channels:
+  - https://conda.anaconda.org/conda-forge
+  - https://conda.anaconda.org/plotly
+  - http://repo.anaconda.com/pkgs/main
 
 specs:
   - bzip2
   - conda {{ version.split("-")[0] }}
   - fasteners
+  - git
```

Comment:
@eugeneswalker Is git really required? I saw you were installing it over on https://github.com/eugeneswalker/jupyterlab-taucmdr

FYI, I think you should be able to grab the installer from [here]() and use this packaged Python 3 environment. Python 3 support is nearly there, but for the meantime it would be useful to know if you can utilize the CommanderConda3 build. The only venv you may need to create and stuff you may need to install is for the python2 environment. Hopefully everything you need is packaged in the CommanderConda3 installer aside from the python 2 stuff (which will be going away soon).